Updates/5.0.0-rc7.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\TagManager;
+
+use Piwik\Plugins\TagManager\Template\Tag\MatomoTag;
+use Piwik\Plugins\TagManager\UpdateHelper\NewTagParameterMigrator;
+use Piwik\Updater;
+use Piwik\Updater\Migration;
+use Piwik\Updater\Migration\Factory as MigrationFactory;
+use Piwik\Updates as PiwikUpdates;
+
+/**
+ * Update for version 5.0.0-rc7.
+ */
+class Updates_5_0_0_rc7 extends PiwikUpdates
```

Comment:
@snake14 RC7 is already released, need to change this

Reply:
Yeah. Apparently I took too long to fix this PR. It hadn't been released yet when I created the migration :man_facepalming: